common/src/main/scala/com/gu/media/aws/AwsCredentials.scala

```diff
@@ -0,0 +1,58 @@
+package com.gu.media.aws
+
+import com.amazonaws.auth._
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClientBuilder
+import com.gu.media.Settings
+
+case class AwsCredentials(instance: AWSCredentialsProvider, crossAccount: AWSCredentialsProvider,
+                          upload: AWSCredentialsProvider)
```

Comment:
Would something like kinesisAccount be a better name than crossAccount since most of the time of dev we are just using media service credentials?